parity/configuration.rs

```diff
@@ -541,8 +554,7 @@ impl Configuration {
 
 	pub fn dapps_interface(&self) -> String {
 		match self.args.flag_dapps_interface.as_str() {
-			"all" => "0.0.0.0",
-			"local" => "127.0.0.1",
+			"local" => "localhost",
```

Comment:
Cause there is no meaningful domain name that you will use to access the server. For JSONRPC I''ve added a parameter `--jsonrpc-hosts` which allows you to specify domains, but for Dapps server it would be safer to just use localhost. Do you think it makes sense to introduce `all + --dapps-domains`?